action.yaml

```diff
@@ -39,6 +39,15 @@ runs:
         aws-region: eu-west-1
         role-to-assume: ${{ inputs.guActionsStaticSiteRoleArn }}
 
+    - uses: actions/setup-go@v3
+        with:
+          go-version: "1.18"
```

Comment:
1.19 - now available!

Reply:
I've matched this to the version referenced in `go.mod` suggest bumping them together separately.